week1/Program.cs

```diff
@@ -0,0 +1,81 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
+
+app.MapGet("/task1", (string str) =>
+{
+    string reversed = "";
+
+    foreach (char c in str)
+    {
+        reversed = c + reversed;
+

```

Comment:
Good job on the first assignment. You have only this extra line here. I find this natural way of solving it.

As mentioned in the classroom, .NET is full of built-in methods, and classes that help you do things. You could write it shorter with 

```
string reversed = string.Join("", str.Reverse()); // str.Reverse() will return array of characters in reverse order and string.Join will join all characters from str.Reverse()
return reversed;
```